### PR TITLE
BITMAKER-2142 estela-web: Implement "In Projects/Scheduled Jobs"

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

estela-web: Implement "In Projects/Scheduled Jobs"

# Issue

https://tasks.bitmaker.dev/issues/2142

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by raymond1242** [comment]
@Adriana618 This is ready to review

File: estela-web/src/components/CronJobDetailPage/index.tsx [hunk 1]
```diff
@@ -332,69 +511,855 @@ export class CronJobDetailPage extends Component<RouteComponentProps<RouteParams
...
+                                            </Text> */}
+                                            {/* <Text
                                                 editable={{
                                                     tooltip: "click to edit text",
                                                     onChange: this.handleInputChange,
                                                 }}
                                             >
                                                 <b>Schedule:</b>&nbsp; {schedule}
-                                            </Text>
+                                            </Text> */}
```
Comment (Adriana618) [inline estela-web/src/components/CronJobDetailPage/index.tsx:1359]: Remove all comments.

File: estela-web/src/components/CronJobDetailPage/index.tsx [hunk 2]
```diff
@@ -160,6 +307,25 @@ export class CronJobDetailPage extends Component<RouteComponentProps<RouteParams
...
+                        tableStatus[1] = false;
+                    }
+                    if (completedJobs.length === 0) {
+                        tableStatus[2] = false;
+                    }
+                    if (errorJobs.length === 0) {
+                        tableStatus[3] = false;
+                    }
+                    const weekDays = this.state.weekDays;
+                    weekDays[moment().day() % 7] = true;
```
Comment (Adriana618) [inline estela-web/src/components/CronJobDetailPage/index.tsx:328]: It would be better to run a separate query for each type of Status.

**Review by Adriana618 — CHANGES_REQUESTED** [review]

File: estela-web/src/components/CronJobDetailPage/index.tsx (hunk 1 above)
Comment (raymond1242) [reply estela-web/src/components/CronJobDetailPage/index.tsx:1359]: Done

**Comment by raymond1242** [comment]
@Adriana618 I fixed the comment you did

**Review by Adriana618 — APPROVED** [review]
Looks good.

**Pull request merged**